internal/crypto/src/tests/raw_signature/rust_native/validators.rs

```diff
@@ -33,7 +30,7 @@ fn es256() {
 }
 
 #[test]
-#[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
+#[cfg(any(target_arch = "wasm32", feature = "rust_native_crypto", test))]
```

Comment:
Could rust_native_crypto be used outside a wasm32 architecture? I'm wondering if we need to specify wasm32 or just use the rust_native_crypto feature here?

Reply:
I did not know who might be depending on that feature being present, like Verify.